tests/e2e/tests/content-manager/history.spec.ts

```diff
@@ -281,6 +280,35 @@ describeOnCondition(edition === 'EE')('History', () => {
       await expect(page.getByText('titleRename')).toBeVisible();
       await page.getByRole('status').getByText('New field');
     });
+
+    test('A user should be able to restore a history version', async ({ page }) => {
+      await page.getByRole('link', { name: 'Content Manager' }).click();
+      await page.getByRole('link', { name: /Create new entry/, exact: true }).click();
+      await page.waitForURL(ARTICLE_CREATE_URL);
+
+      const titleInput = await page.getByRole('textbox', { name: 'title' });
+      // Create an initial entry to also create an initial version
+      await titleInput.fill('Being from Kansas');
+      await page.getByRole('button', { name: 'Save' }).click();
+      await page.waitForURL(ARTICLE_EDIT_URL);
+      // Update to create another version
+      await titleInput.fill('Being from Florida');
+      await page.getByRole('button', { name: 'Publish' }).click();
```

Comment:
I believe you mean to click the save button here, not publish

Reply:
Yeah... I did this intentionally 🙃 should have mentioned it. The save button for some reason is not being enabled when I fill the text input a second time. I saw this [issue on playwright](https://github.com/microsoft/playwright/issues/27564) where they indicate it is related to [hydration](https://playwright.dev/python/docs/navigations#hydration). I'm not sure if that's the cause for us but also not sure it's an issue with playwright. Wondering too if this is maybe caused by a similar issue:

https://github.com/strapi/strapi/blob/955bc0269eee58bb62229db838870577196ded81/tests/e2e/tests/content-manager/history.spec.ts#L31-L37

Happy to investigate just not sure if it should be done here on this PR...